explainer.md

```diff
@@ -86,39 +116,73 @@ Note that as part of this proposal, we are proposing to [remove the requirement]
 
 ### Acknowledging a purchase
 
-Some stores will require that the user acknowledge a purchase once it has succeeded. In this case, the payment response will return a "purchase token" string, which can be used with the `acknowledge` method.
-
-Items that are designed to be purchased multiple times must be acknowledged with the `repeatable` flag. An example of a repeatable purchase is an in-game powerup that makes the player stronger for a short period of time. Once it is acknowledged with the `repeatable` flag, it can be purchased again.
-
-```js
-digitalGoodsService.acknowledge(purchaseToken, 'repeatable');
-```
+Some stores will require that the developer acknowledge a purchase once it has succeeded. In this case, the payment response will return a "purchase token" string, which should be verified and acknowledged by the developer. This should be done using direct API calls from the developer's server to the service provider before granting entitlements. Such a direct developer-to-provider API is not specified as part of the Digital Goods API.
 
-Items that are designed to be purchased once and last permanently in the user’s app must be acknowledged with the `onetime` flag. An example of a one-time purchase is a “remove ads” option. Once acknowledged with the `onetime` flag, the app is expected to remember the user’s purchase and continue providing the purchased capability.
+### Consuming a purchase
 
+Items that are designed to be purchased multiple times usually need to be marked as "consumed" before they can be purchased again by the user. An example of a consumable purchase is an in-game powerup that makes the player stronger for a short period of time. This can be done with the `consume` method:
 
 ```js
-digitalGoodsService.acknowledge(purchaseToken, 'onetime');
+digitalGoodsService.consume(purchaseToken);
 ```
 
-### Checking existing purchases
+It is preferable to use a direct developer-to-provider API to consume purchases, if one is available, in order to more verifiably ensure that a purchase was used up.
```

Comment:
It is preferable not to use the DGAPI? How come and how come we are creating the DGAPI?

Reply:
Unfortunately yes. Client-side code is inherently vulnerable to tampering, unlike direct developer-to-service-provider communication ([more detail here](https://docs.google.com/document/d/1wgmfo1TdtYg5Y69THCMSDIMLYDaNWLN8clYayv7HsqM/edit?resourcekey=0-F0W8BjxzOMjS7CNN8Myacg#heading=h.y3crxg196iu)). Part of the motivation for creating the DGAPI is that direct developer-to-provider APIs don't exist for most of the API surface.